python/tank/descriptor/io_descriptor/factory.py

```diff
@@ -89,9 +89,16 @@ def create_io_descriptor(
     # and it doesn't matter where we are fetching it from. If <core appstore v1.2.3>
     # is available in multiple different locations on disk, the content of each location
     # should be identical
-    if descriptor_uri in g_cached_instances:
-        # cache hit
-        return g_cached_instances[descriptor_uri]
+    # FIXME: Disabling caches as it causes issues when caching bundles from an installed configuration
```

Comment:
Hum... I'm still not sure however installed is the right word. You could imagine someone taking over the pipeline configuration with a plugin id and set the paths. So installed doesn't necessarily mean the "old way of doing things", it simply means using the *_paths. The old way of doing things is using the *_paths AND not setting plugin_id.

Maybe we need to revisit the terminology and update the documentation accordingly?